ooasp/encodings/ooasp_guess.lp

```diff
@@ -1,6 +1,17 @@
 % Copyright (c) 2022 Siemens AG Oesterreich
```

Comment:
```suggestion
% Copyright (c) 2022-2024 Siemens AG Oesterreich
```
Please remember to update and add this also in other files (that are touched or newly created)

Reply:
@rtaupe would you like me to do this in a separate commit?

Reply:
That's not necessary.